@navikt/core/css/darkside/date.darkside.css

```diff
@@ -1,222 +1,206 @@
+  }
 
-.navds-date .rdp-weeknumber {
-  font-size: var(--a-font-size-small);
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  width: 2rem;
-  height: 2rem;
-  border-radius: var(--a-border-radius-medium);
-  margin: var(--a-spacing-2);
-  color: var(--a-text-subtle);
-}
+  .rdp-weeknumber {
+    font-size: var(--ax-font-size-small);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 2rem;
+    height: 2rem;
+    border-radius: var(--ax-border-radius-medium);
+    margin: var(--ax-spacing-2);
+    color: var(--ax-text-subtle);
+  }
 
-.navds-date .rdp-weeknumber.rdp-button {
-  width: 2rem;
-  height: 2rem;
-  box-shadow: 0 0 0 1px var(--a-border-default);
-  color: var(--a-text-subtle);
-  font-size: var(--a-font-size-small);
-}
+  .rdp-weeknumber.rdp-button {
+    width: 2rem;
+    height: 2rem;
+    box-shadow: 0 0 0 1px var(--ax-border-default);
+    color: var(--ax-text-subtle);
+    font-size: var(--ax-font-size-small);
+  }
 
-.navds-date .rdp-weeknumber.rdp-button:active {
-  background-color: var(--a-surface-action-active);
-  color: var(--a-text-on-action);
-  box-shadow: none;
-}
+  .rdp-weeknumber.rdp-button:active {
+    background-color: var(--ax-bg-accent-strong-pressed);
+    color: var(--ax-text-accent-contrast);
+    box-shadow: none;
+  }
 
-.navds-date__caption__month .navds-select__container select {
-  text-transform: capitalize;
-}
+  .rdp-button {
+    all: unset;
+    display: block;
+    width: 2.5rem;
+    height: 2.5rem;
+    text-align: center;
+    border-radius: var(--ax-border-radius-medium);
+  }
 
-.navds-date .rdp-button {
-  all: unset;
-  display: block;
-  width: 2.5rem;
-  height: 2.5rem;
-  text-align: center;
-  border-radius: var(--a-border-radius-medium);
-}
+  .rdp-day_range_start {
+    border-radius: var(--ax-border-radius-xlarge) var(--ax-border-radius-medium) var(--ax-border-radius-medium)
+      var(--ax-border-radius-xlarge);
+  }
 
-.navds-date .rdp-day_range_start {
-  border-radius: var(--a-border-radius-xlarge) var(--a-border-radius-medium) var(--a-border-radius-medium)
-    var(--a-border-radius-xlarge);
-}
+  .rdp-day_range_end:not(.rdp-day_range_start) {
+    border-radius: var(--ax-border-radius-medium) var(--ax-border-radius-xlarge) var(--ax-border-radius-xlarge)
+      var(--ax-border-radius-medium);
+  }
 
-.navds-date .rdp-day_range_end:not(.rdp-day_range_start) {
-  border-radius: var(--a-border-radius-medium) var(--a-border-radius-xlarge) var(--a-border-radius-xlarge)
-    var(--a-border-radius-medium);
-}
+  .rdp-day_range_start.rdp-day_range_end {
+    border-radius: var(--ax-border-radius-xlarge);
+  }
 
-.navds-date .rdp-day_range_start.rdp-day_range_end {
-  border-radius: var(--a-border-radius-xlarge);
-}
+  .navds-date__caption__month,
+  .navds-date__caption__year,
+  .rdp-button {
+    &:focus-within {
+      outline: 2px solid var(--ax-border-focus);
+      outline-offset: 2px;
```

Comment:
```suggestion
      outline-offset: 1px;
```

Think 1px offset looks better and avoids overlap
![Screenshot 2024-11-25 at 12 39 10](https://github.com/user-attachments/assets/77b85ebc-c1a5-45b1-bd22-41d1e4a8bca2)
![Screenshot 2024-11-25 at 12 39 25](https://github.com/user-attachments/assets/c45f8dd4-2f27-458d-b012-f1d23dfb3017)